core/Pos/Binary/Crypto.hs

```diff
@@ -142,7 +142,7 @@ secretKeyLength, publicKeyLength, signatureLength, chainCodeLength,
     encryptedKeyLength, passphraseLength :: Int
 secretKeyLength = 32
 publicKeyLength = 32
-encryptedKeyLength = 96
+encryptedKeyLength = 128
```

Comment:
@neongreen which constants do we want in `master` ? I guess its expected to have these change in `master`?